### DISCUSSION TITLE
Best Practices for Strapi Collection, Type User

### DISCUSSION THREAD

**Discussion by onsombal** [strapi/strapi#7820] [Legacy Topics]
I'm using "strapi": "3.1.4" and MongoDB-Atlas for my backend.

Is there a reason why the default Collection for the Strapi **User Type** isn't presented in the Strapi server installation's **/api** directory along with the custom Collection Types that were created? 

The User Type is presented in the playground Docs, e.g. -- user(id: ID!): UsersPermissionsUser.

Is there a best practice about extending or not extending the User Type with additional fields?

Thanks...

**Comment by derrickmehaffy** [comment]
Certain plugins have their own models which don't have the normal CRUD actions (most do but they are highly customized for that specific plugin)

Not all models are stored under the `./api/*` path, only user created ones. For extending you will want to see this FAQ entry: https://strapi.io/documentation/v3.x/getting-started/troubleshooting.html#how-do-i-customize-a-plugin

By default the Content-Type builder will handle the relations building and extending of the User model settings on it's own (you will see the `User.settings.json` under `./extensions/users-permissions/models/User.settings.json` created for you when a new relation is added.
